Migration to Bootstrap 5 for HumHub 1.18

yurabakhtin opened the pull request.

marc-farre commented:
@yurabakhtin Thanks! Should I review and test it?

yurabakhtin commented:
> Thanks! Should I review and test it?

@marc-farre Please wait, I just started it, I will write when it will be ready.

marc-farre commented:
@yurabakhtin FYI, I have merged master: https://github.com/humhub/custom-pages/pull/435/commits/1d043dab298455f0b9fe3d9f0c9ea6723b3b93e7

And in commit https://github.com/humhub/custom-pages/pull/435/commits/b57c3a1674085e9839d851ff3dabc141ef0d4ef5 , I have replaced Labels with Badges:
- Do we still need the `small` tags around the badges?
- Can you check the new badges are displayed correctly?

yurabakhtin commented:
> FYI, I have merged master: [1d043da](https://github.com/humhub/custom-pages/commit/1d043dab298455f0b9fe3d9f0c9ea6723b3b93e7) and now Codeception tests are all working on GitHub too.

@marc-farre Thank you, probably the test error was only on the branch `fix/bs5-tests` that we changed to correct `bs5` yesterday https://github.com/humhub/actions/commit/fb91b1d670f549e9dd4926f6da968f1a0fdf0d1a.

> * Do we still need the `small` tags around the badges?

No need, I removed them in the commit https://github.com/humhub/custom-pages/pull/435/commits/2c922f9bad721e278dba2647ec62c95f7c075053.

> * Can you check the new badges are displayed correctly?

I have checked, they looks correctly.
